XenAdmin/Controls/PvsCacheStorageRow.cs

```diff
@@ -199,6 +202,16 @@ private void comboBoxCacheSr_SelectedIndexChanged(object sender, EventArgs e)
             }
             SomethingChanged(this, e);
         }
+
+        private void pictureBoxInfo_Click(object sender, EventArgs e)

```

Comment:
I think the tooltip would look better if it was displayed on the disabled controls, not just once the (i) icon is clicked.